### ISSUE TITLE
Empty metadata fields fail to hide or display emptyText

### ISSUE THREAD

**Issue by HitmanInWis** [Adobe-Marketing-Cloud/asset-share-commons#311]
**Describe the bug**
Some metadata fields that resolve to an empty value do not display the emptyText value.

**Environment**
* AEM Version: 6.4
* Asset Share Commons Version: 1.6.0

**To Reproduce**
1. Install the default Asset Share Commons
1. Go to /content/dam/asset-share-commons/en/public/pictures/igor-son-320878.jpg/jcr:content/metadata
1. Remove properties "tiff:ImageLength" and "tiff:ImageHeight"
1. Add empty property "dc:description"
1. Open up http://localhost:4502/editor.html/content/asset-share-commons/en/dark/details/image.html/content/dam/asset-share-commons/en/public/pictures/igor-son-320878.jpg
1. Notice that both Description and Resolution are still displayed, even though empty.
1. Configure either metadata component with Empty Text
1. Notice that the emptyText still does not display.

**Expected behavior**
When these metadata fields are empty the field should be hidden from the rendered UI.

**Additional context**
I believe the issues is in https://github.com/Adobe-Marketing-Cloud/asset-share-commons/blob/develop/core/src/main/java/com/adobe/aem/commons/assetshare/components/details/impl/MetadataImpl.java - line 155

```
return combinedProperties.get(getPropertyName()) == null;
```
should instead be
```
return StringUtils.isBlank(return combinedProperties.get(getPropertyName()));
```

**Comment by davidjgonzalez** [comment]
You wanna PR/commit since you did all the heavy work? :)  If not easy enough to do it on this end.

**Comment by HitmanInWis** [comment]
sure thing, I think I can handle this one ;)

**Comment by davidjgonzalez** [comment]
merged to develop and slated for 1.6.10
